src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
Hi @trxcllnt I think that I've found the issue here and it is related to the `redux-falcor` component; I will update this PR and try to explain better what I mean by that. Thanks.